### PR TITLE
chore: add issue templates

### PR DESCRIPTION
**Pull request by yordis**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    validations:
+      required: true
+
+  - type: input
+    id: erlang-version
+    attributes:
+      label: Erlang version
+      description: Use `elixir -v` to find the Erlang version.
+    validations:
+      required: true
```
Comment (whatyouhide) [inline .github/ISSUE_TEMPLATE/bug.yml:19]: Can we merge these into a single field, in the interest of streamlining the process as much as possible?

```suggestion
  - type: input
    id: elixir-and-otp-version
    attributes:
      label: Elixir and Erlang/OTP versions
      description: Paste the output of `elixir -v` here.
    validations:
      required: true
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  - type: input
+    id: os
+    attributes:
+      label: Operation System
+      description: The operation system
```
Comment (whatyouhide) [inline .github/ISSUE_TEMPLATE/bug.yml:25]:
```suggestion
      label: Operating System
      description: The operating system that this issue is happening on.
```

**Comment by yordis** [comment]
@whatyouhide hopefully the single field doesn't show the info in a weird way when people paste it. Let see.

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+name: 🐞 Bug Report
+description: Tell us about something that's not working the way we (probably) intend.
+labels: ["Kind:Bug", "State:Triage"]
```
Comment (josevalim) [inline .github/ISSUE_TEMPLATE/bug.yml:3]:
```suggestion
```

In Elixir, everything untagged is untriaged. Maybe having a explicit state is good, but I want to try that within Ecto first. :)

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+blank_issues_enabled: false
+
+contact_links:
+  - name: Discuss proposals
+    url: https://groups.google.com/g/elixir-lang-core
+    about: Discuss proposals
+  - name: Ask Questions
+    url: https://elixirforum.com/
+    about: Ask and answer questions here.
```
Comment (josevalim) [inline .github/ISSUE_TEMPLATE/config.yml:9]:
```suggestion
  - name: Ask questions
    url: https://elixirforum.com/
    about: Ask and answer questions on ElixirForum.
```

**Comment by eksperimental** [comment]
I am not too sure "Bug Report" describes all the situations,
as the description says "Tell us about something that's not working the way we (probably) intend."
I think "Report an Issue"  as we used before describes better this, as a generated warning, a typo, an unclear error message, an optimization, and so on are not bugs per-se.

Well, those are my 2 cents.

**Comment by josevalim** [comment]
@eksperimental i agree, can you please send a PR? :)

**Comment by eksperimental** [comment]
@josevalim sure.

**Pull request merged**